lib/rules/base.js

```diff
@@ -69,6 +73,18 @@ module.exports = class Base {
     return this._configResolver.editorConfig();
   }
 
+  get _moduleName() {
+    if (!loggedModules.has(this.ruleName)) {
+      this._console.log(
+        'The `_moduleName` property is deprecated. Please use `_filePath` instead.'
```

Comment:
Please include the rule name in the message, without it this is very hard to track down for folks.